Ollivanders/src/net/pottercraft/ollivanders2/Ollivanders2.java

```diff
@@ -397,7 +397,13 @@ private void initConfig ()
       }
 
       //
-      // apparate locations, when true apparate will only work to named locations, not by coordinates
+      // apparate distance, when set, will limit apparate distance in the X and Z directions
+      //
+      APPARATE.maxApparateDistance = getConfig().getInt("maxApparateDistance");
```

Comment:
Read the config in the same place where we read all other config so we only need to read it once during plugin enable.